controllers/openstackcluster_controller.go

```diff
@@ -193,7 +193,7 @@ func (r *OpenStackClusterReconciler) reconcileDelete(ctx context.Context, scope
 
 	// Cluster is deleted so remove the finalizer.
 	controllerutil.RemoveFinalizer(openStackCluster, infrav1.ClusterFinalizer)
-	scope.Logger().Info("Reconciled Cluster delete successfully")
+	scope.Logger().Info("Reconciled Cluster deleted successfully")
```

Comment:
Aside: you've made me read this differently. I had always read it as:

```
verb(Reconconciled) noun(Cluster delete) adverb(successfully)
```

But you're reading it as:

```
participle adjective(Reconciled) noun(Cluster) verb(deleted) adverb(successfully)
```

which is equally valid. English is insane 🤦

This is good, btw. I'm not asking you to change it.